examples/basic-webpack/index.jsx

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+var ExampleApplication = React.createClass({
+  render: function() {
+    var elapsed = Math.round(this.props.elapsed  / 100);
+    var seconds = elapsed / 10 + (elapsed % 10 ? '' : '.0' );
+    var message = 'React has been successfully running for ' + seconds + ' seconds.';
+
+    return (<p>{message}</p>);
+  }
+});
+
+var start = new Date().getTime();
+
+setInterval(function() {
```

Comment:
Just curious -- instead of calling `ReactDOM.render` repeatedly, why not using `setState` inside the component? IMO calling `ReactDOM.render` multiple times is an anti-pattern.